### PR TITLE
PMM-9610-DB-clusters-after-PMM-upgrade

### PR DESCRIPTION
**Pull request by beatahandzelova**
2.30.0 > 2.31.0 https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-dbaas-upgrade-tests/detail/pmm2-dbaas-upgrade-tests/294/pipeline/
2.31.0 > 2.32.0 https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-dbaas-upgrade-tests/detail/pmm2-dbaas-upgrade-tests/296/pipeline/

### CODE REVIEW

**Review by peterSirotnak — DISMISSED** [review]

File: tests/DbaaS/pages/dbaasActionsPage.js [hunk 1]
```diff
@@ -5,13 +5,13 @@ const assert = require('assert');
 
 module.exports = {
 
-  async checkActionPossible(actionName, actionPosibilty) {
+  async checkActionPossible(actionName, actionPosibilty, dbclusterName) {
```
Comment (vyarosh) [inline tests/DbaaS/pages/dbaasActionsPage.js:8]: Please add JS doc with description and explanation of the arguments(please include complete list of values for the first argument). Also as I see second argument is boolean please apply any naming convention to make it obvious the variable/argument is boolean. I assume this argument is `expectedResult` is it correct?

File: tests/DbaaS/pages/dbaasActionsPage.js [hunk 2]
```diff
@@ -72,10 +72,11 @@ module.exports = {
     }
   },
 
-  async deleteXtraDBCluster(dbClusterName, k8sClusterName) {
+  async deleteXtraDBCluster(dbClusterName, k8sClusterName, deleteCompleted = true) {
```
Comment (vyarosh) [inline tests/DbaaS/pages/dbaasActionsPage.js:75]: Please use the CamelCase for the functions names. See: https://github.com/percona/pmm-ui-tests/blob/main/CONTRIBUTING.md#naming-conventions

File: tests/verifyDBaaSUpgrade_test.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    I.amOnPage('graph/dbaas/dbclusters');
+    I.waitForText(active_state, 10, dbaasPage.tabs.dbClusterTab.fields.clusterTableRow(pxc_cluster_name));
+    I.waitForText(active_state, 10, dbaasPage.tabs.dbClusterTab.fields.clusterTableRow(psmdb_cluster_name));
+
+    await dbaasPage.verifyLogPopup(33, psmdb_cluster_name);
+    await dbaasPage.verifyLogPopup(6, pxc_cluster_name);
+  }
+);
+
+const pxcDBClusterDetails = new DataTable(['namespace', 'clusterName', 'node']);
```
Comment (vyarosh) [inline tests/verifyDBaaSUpgrade_test.js:48]: Please apply apply camelCase to given constant name

File: tests/DbaaS/pages/dbaasActionsPage.js (hunk 2 above)
Comment (beatahandzelova) [reply tests/DbaaS/pages/dbaasActionsPage.js:80]: no related to my changes, will fix all function names later in refactoring task

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**